eng/pipelines/common/templates/wasm-debugger-tests.yml

```diff
@@ -4,6 +4,7 @@ parameters:
   isWasmOnlyBuild: false
   browser: 'chrome'
   shouldContinueOnError: false
+  runOnlyOnWasmPipelines: false
```

Comment:
I'm confused as to why this would want to run on non-wasm pipelines. Are the wasm debugger tests not a wasm pipeline?

Reply:
it should really be named `runOnlyOnWasmOnlyPipelines` or such!

Reply:
So, it should run only on runtime-wasm* pipelines which have only wasm builds, and are run manually.